src_assets/common/assets/web/config.html

```diff
@@ -872,41 +872,73 @@ <h1 class="my-4">Configuration</h1>
-          <option value="constqp">constqp -- constant qp mode</option>
-          <option value="vbr">vbr -- variable bitrate</option>
-          <option value="cbr">cbr -- constant bitrate (default)</option>
+        <label for="nvenc_twopass" class="form-label">Two-pass mode</label>
+        <select id="nvenc_twopass" class="form-select" v-model="config.nvenc_twopass">
+          <option value="disabled">Disabled (fastest, not recommended)</option>
+          <option value="quarter_res">Quarter resolution (faster, default)</option>
+          <option value="full_res">Full resolution (slower)</option>
         </select>
+        <div class="form-text">Adds preliminary encoding pass.<br>
+          This allows to detect more motion vectors, better distribute bitrate across the frame and more strictly adhere to
+          bitrate limits.<br>
+          Disabling it is not recommended since this can lead to occasional bitrate overshoot and subsequent packet loss.
+        </div>
       </div>
-      <div class="mb-3">
-        <label for="nv_coder" class="form-label">NVENC Coder (H264)</label>
-        <select id="nv_coder" class="form-select" v-model="config.nv_coder">
-          <option value="auto">auto -- let ffmpeg decide (default)</option>
-          <option value="cabac">cabac -- context adaptive binary arithmetic coding - higher quality</option>
-          <option value="cavlc">cavlc -- context adaptive variable-length coding - faster decode</option>
-        </select>
+      <div class="accordion">
+        <div class="accordion-item">
+          <h2 class="accordion-header">
+            <button class="accordion-button" type="button" data-bs-toggle="collapse"
+              data-bs-target="#panelsStayOpen-collapseOne">
+              Miscellaneous options
+            </button>
+          </h2>
+          <div id="panelsStayOpen-collapseOne" class="accordion-collapse collapse show"
+            aria-labelledby="panelsStayOpen-headingOne">
+            <div class="accordion-body">
+              <div class="mb-3">
+                <label for="nvenc_realtime_hags" class="form-label">Use realtime priority in hardware accelerated gpu scheduling</label>
+                <select id="nvenc_realtime_hags" class="form-select" v-model="config.nvenc_realtime_hags">
+                  <option value="disabled">Disabled</option>
+                  <option value="enabled">Enabled (default)</option>
+                </select>
+                <div class="form-text">Currently NVIDIA drivers may freeze in encoder when
+                  <a href="https://devblogs.microsoft.com/directx/hardware-accelerated-gpu-scheduling/">HAGS</a>
+                  is enabled, realtime priority is used and VRAM utilization is close to maximum.<br>
+                  Disabling this option lowers the priority to high sidestepping the freeze, but capture performance may suffer
+                  when GPU is heavily loaded.
```

Comment:
The second sentence there is a bit hard to understand. Maybe something like: "Disabling this option lowers the priority to high, sidestepping the freeze at the cost of reduced capture performance when the GPU is heavily loaded."

Reply:
Copy-pasted, thank you.